<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1051</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电商交易平台</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>请先登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passwordButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="discountButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打折</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="managegoodsButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>商品管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="managebalanceButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>余额管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addgoodsButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>620</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加商品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelloginButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>70</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
